src/bsoncxx/view_or_value.hpp

```diff
@@ -118,7 +118,7 @@ class view_or_value {
         return _view;
     }
 
-   private:
+   protected:
```

Comment:
I'm fine with it, please though add a TODO: make these members private. I don't want to hold up this change over it.